### ISSUE TITLE
bootstrap gulp task Issue

### ISSUE THREAD

**Issue by adil-soubki** [yan-foto/neutron#15]
#### My Configuration

I am running OSX Yosemite 10.10.4 with the latest prebuilt binaries of Electron. 
#### Issue

Hi I noticed you added a bootstrap command. I am having some trouble with it. I checked out a clean version of the repo and ran the following commands. Any idea what's going on?

```
$ git clone https://github.com/yan-foto/neutron.git
$ npm run bootstrap
npm ERR! Darwin 14.4.0
npm ERR! argv "node" "/usr/local/bin/npm" "run" "bootstrap"
npm ERR! node v0.12.2
npm ERR! npm  v2.13.5

npm ERR! missing script: bootstrap
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Volumes/hrt/projects/neutron_reports/npm-debug.log
```

I also tried

```
$ gulp bootstrap
[14:24:46] Failed to load external module babel-core/register
[14:24:46] Failed to load external module babel/register
[14:24:46] Local gulp not found in /Volumes/hrt/projects/neutron_reports
[14:24:46] Try running: npm install gulp
```

Gulp is installed both locally and globally.

**Comment by yan-foto** [comment]
Bootstrap is a gulp task and not an npm script. As it states in the first line, you will need `babel-core` installed. Any luck?

**Comment by adil-soubki** [comment]
Sorry, thanks.

**Comment by yan-foto** [comment]
Please be a bit more patient before filing issues.

**Comment by adil-soubki** [comment]
Yes, sorry. 

I guess I don't fully understand the point of the bootstrap command. In my experience I had to go down the list and `npm install` every dependency until it worked (eventually I just let the package.json do the work). Then once I did get it to work, it would only run through `gulp` and not through `npm run`. At that point it appears it only checked the modules that were installed already. I guess I would just like to know if I am using this feature correctly and if so was that the intended result?

I did not intend to waste your time.
